oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/source/DefaultScopeDefine.java

```diff
@@ -106,6 +107,10 @@ public class DefaultScopeDefine {
     public static final int ZIPKIN_SERVICE_SPAN = 52;
     public static final int ZIPKIN_SERVICE_RELATION = 53;
     public static final int PROCESS_RELATION = 54;
+    public static final int CACHE_READ_ACCESS = 55;
+    public static final int CACHE_WRITE_ACCESS = 56;
+    public static final int CACHE_SLOW_WRITE = 57;
+    public static final int CACHE_SLOW_READ = 58;
```

Comment:
These four should be merged into two. The source should include read/write as an attribute rather than using two sources.